configure.ac

```diff
@@ -93,6 +93,22 @@ case "${host_os}" in
 	;;
 	darwin*)
 		ISDARWIN=1
+
+		AC_CHECK_PROG([BREW],brew, brew)
+		if test x$BREW = xbrew; then
+			dnl add default homebrew paths
+
+			icu4c_prefix=$($BREW --prefix icu4c)
+			export PKG_CONFIG_PATH="$icu4c_prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
+
+			python3_prefix=$(dirname "$(find "$($BREW --prefix python3)/" -name python3.pc)")
+			export PKG_CONFIG_PATH="$python3_prefix:$PKG_CONFIG_PATH"
+
+			if test -z "$USE_SYSTEM_OPENSSL" -o "$USE_SYSTEM_OPENSSL" = "false"; then
```

Comment:
Ok. But please add some ifs for ssl/python/icu: if they are installed in homebrew, put some message that using version from homebrew.
Makes debugging easier.